src/lib/data/alignment.js

```diff
@@ -61,41 +62,50 @@ export default class Alignment {
     return this.target.alignedText
   }
 
+  shouldStartNewAlignmentGroup (token) {
+    return !this.currentAlignmentGroup || !this.currentAlignmentGroup.couldBeAdded(token)

```

Comment:
this business logic needs to be extended to support the scenario where it's the start word of the alignment group that is clicked  on.  In that case, the current alignment group should be finished and a new one should *not* be automatically be started.  Otherwise the current token ends up in many alignment groups (which is breaking the business logic which says a token cannot belong in more than one group at at time -- i.e. see the merge group scenario)

Reply:
oh, I missed the correct workflow. I fixed the conditions to support the requirements:
```
Workflow 1 New Alignment starting from Source
- Click on a word in the source text
- Click on one or more words in the translation text
- Click on the aligned word in the source text to save

Workflow 2 New Aignment starting from Translation
- Click on a word the translation text
- Click on one or more words in the source text
- Click on the aligned word in the translation text to save
```

Reply:
ok, this is better now. I think we still need to implement the merge scenario.  I've updated #24 to include a note about that.